api/src/main/java/io/strimzi/api/kafka/model/KafkaConnect.java

```diff
@@ -185,4 +185,10 @@ public String toString() {
     public void setAdditionalProperty(String name, Object value) {
         this.additionalProperties.put(name, value);
     }
+
+    @Override
+    @Description("Rack awareness configuration.")
+    public Rack getRack() {
+        return spec.getRack();
+    }
```

Comment:
This seems wrong. This is a property of the spec not of the CRD.

Reply:
OK, thanks.